com.unity.toonshader/Documentation~/Known-issue.md

```diff
@@ -1,15 +1,16 @@
 # Known Issues
-
 ## When using AOV Image Sequence Recorder
 
-The **Unity Toon Shader** [Outline](Outline.md) isn't automatically disabled when recording with  [AOV Image Sequence Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@4.0/manual/RecorderAOV.html), Please disable [Outline](Outline.md#outline) in Material Inspectors when outputting AOV Images.
+The **Unity Toon Shader** [Outline](Outline.md) isn't automatically disabled when recording with [AOV Image Sequence Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@4.0/manual/RecorderAOV.html). Please disable [Outline](Outline.md#outline) in Material Inspectors when outputting AOV Images.
 
 ## When using Universal Render Pipeline
 
 When using [Universal Render Pipeline](https://docs.unity3d.com/Manual/com.unity.render-pipelines.universal.html), you need to set [Depth Priming Mode](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@16.0/manual/urp-universal-renderer.html#rendering) disabled to show [outline](Outline.md).
 
+<canvas class="image-comparison" role="img" aria-label="A toon-shaded model of a human, and the Inspector window for the Univeral Renderer Data asset. On the left, Depth Priming is set to Auto and the model has no outline. On the right, Depth Priming is set to Disabled and model has an outline.">
+    <img src="images/DepthPrimingModeAuto.png" title="Depth Priming Mode set to Auto">
```

Comment:
Hi, I use Visual Studio code to check preview of the pages, but I can't see this part with it. What kind of tool do you recommend to preview theese parts?